tests/test_pipelines_common.py

```diff
@@ -322,6 +323,19 @@ def data(n: int):
             results.append(out)
         self.assertEqual(len(results), 10)
 
+    @require_torch
+    def test_unbatch_attentions_hidden_states(self):
+        model = DistilBertForSequenceClassification.from_pretrained(
+            "Narsil/tiny-distilbert-sequence-classification", output_hidden_states=True, output_attentions=True
```

Comment:
Can this leverage the one from `hf-internal-testing`, or is there a reason you favor this one?

Reply:
There is not `tiny-distilbert` in `hf-internal-testing`, only the random one.
Happy to upload it if you want (as well as all the other `Narsil` specific tests that probably all belong in `hf-internal-testing`.

For instance this one was just copied from the `text-classification` tests.

Reply:
Right, but the random has random weights but actually has the weights so it wouldn't be initialized randomly every time you load it in memory.

It shouldn't be an issue here given that you're only checking the length of outputs, right?

Reply:
Well I am using these also in other tests, are you ok if I merge this PR, then open another PR to remove `Narsil/` models from a bit everywhere (some check actual output values)